recipes-openxt/initrdscripts/initramfs-xenclient/init.sh

```diff
@@ -169,10 +169,10 @@ tpm_setup() {
     insmod /lib/modules/$(uname -r)/kernel/drivers/char/tpm/tpm.ko
     insmod /lib/modules/$(uname -r)/kernel/drivers/char/tpm/tpm_tis.ko
     mknod /dev/tpm0 c 10 224
-    echo -n "initramfs measuring $ROOT_DEVICE: "
```

Comment:
i am good with letting them know, just prefer we didn't send the hash. This is how i queued up the next iteration.